jekyll/_cci2/server-3-operator-extending-internal-volumes.adoc

```diff
@@ -156,17 +156,51 @@ Filesystem      Size  Used Avail Use% Mounted on
 
 As we can see, the size of our directories has been increased.
 
+When running through these steps, if you find, as expected, that the new pods **do** show the resized volumes, it is still worth running the `kubectl describe` commands shown below to check. In some instances the resize will fail, but the only way to know is by viewing an event in the output from `kubectl describe`.
 
-NOTE: If you find that after following these steps, the disk size allocated to your data directories have not increased, then you may need to restart your database pods. This, however, will cause some downtime of 1-5mins as the databases restart. You can use the commands below to restart your databases.
+For postgres
+[source,bash]
+----
+kubectl describe pvc data-postgresql-0
+----
+
+For mongodb
+[source,bash]
+----
+kubectl describe pvc datadir-mongodb-0
+----
+
+success looks like:
+```
+Events:
+Type    Reason                      Age   From     Message
+
+   ------                      ----  ----     -------
+Normal  FileSystemResizeSuccessful  19m   kubelet  MountVolume.NodeExpandVolume succeeded for volume "pvc-b3382dd7-3ecc-45b0-aeff-45edc31f48aa"
```

Comment:
@ryanwohara this formatting looks totally weird but I copied it from the jira ticket. Any ideas how I can clean it up, or is this correct?